src/main/java/com/google/api/codegen/transformer/ApiMethodTransformer.java

```diff
@@ -202,7 +207,7 @@ private void setFlattenedMethodFields(
     for (Field field : fields) {
       params.add(generateRequestObjectParam(context, field));
     }
-    methodViewBuilder.methodParams(params);
+    methodViewBuilder.methodParams(params).methodParamsWithExtras(params);
```

Comment:
This doesn't follow the pattern of one setting per line - I was actually a bit confused about this for a little while.